src/userCLI/src/main/kotlin/org/vaccineimpact/orderlyweb/userCLI/AddUser.kt

```diff
@@ -3,18 +3,21 @@ package org.vaccineimpact.orderlyweb.userCLI
 import org.vaccineimpact.orderlyweb.db.OrderlyUserRepository
 import org.vaccineimpact.orderlyweb.models.UserSource
 
-fun addUser(options: Map<String, Any>): String
+fun addUsers(options: Map<String, Any>): String
 {
-    val userEmail = options["<email>"].getStringValue()
+    val userEmails = options["<email>"] as List<*>
     val userRepo = OrderlyUserRepository()
 
-    return if (userRepo.getUser(userEmail) == null)
-    {
-        userRepo.addUser(userEmail, "unknown", "unknown", UserSource.CLI)
-        "Saved user with email '$userEmail' to the database"
-    }
-    else
-    {
-        "User with email '$userEmail' already exists; no changes made"
+    return userEmails.joinToString("\n") {
+        val email = it.getStringValue()
+        if (userRepo.getUser(email) == null)
+        {
+            userRepo.addUser(email, "unknown", "unknown", UserSource.CLI)
```

Comment:
Should we make it possible to import existing Montagu users with their usernames / display names as well as email addresses? I think we're currently just displaying the user's id (email) in the Logged In control so we're not actually using display names at the moment. But presumably we will want them at some stage!

Reply:
Once they first log in their username will get updated straight away, so I don't think there's any need.